tests/suites/1_stateful/02_query/02_0004_attach_table.sh

```diff
@@ -4,11 +4,25 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 . "$CURDIR"/../../../shell_env.sh
 
 echo "drop table if exists table_from;" | $BENDSQL_CLIENT_CONNECT
+echo "drop table if exists table_from2;" | $BENDSQL_CLIENT_CONNECT
 echo "drop table if exists table_to;" | $BENDSQL_CLIENT_CONNECT
+echo "drop table if exists table_to2;" | $BENDSQL_CLIENT_CONNECT
+
+query() {
```

Comment:
Can we reuse the `quert` in `shell_env`?

Reply:
done